### PR TITLE
feat: update ci using gh actions

### PR DESCRIPTION
**Pull request by lnahiro**
Update CI by replacing travis with github actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Redis PHP Mock [![Build Status](https://secure.travis-ci.org/M6Web/RedisMock.svg?branch=master)](http://travis-ci.org/M6Web/RedisMock) [![Total Downloads](https://poser.pugx.org/m6web/redis-mock/downloads.svg)](https://packagist.org/packages/m6web/redis-mock)
+# Redis PHP Mock [![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FBedrockStreaming%2FRedisMock%2Fbadge%3Fref%3Dmaster&style=flat)](https://actions-badge.atrox.dev/BedrockStreaming/RedisMock/goto?ref=master) [![Total Downloads](https://poser.pugx.org/m6web/redis-mock/downloads.svg)](https://packagist.org/packages/m6web/redis-mock)
```
Comment (Oliboy50) [inline README.md:1]: @jdecool could you confirm what's the best practice (i.e. the best domain name) for CI badges, please?

I still didn't understand if `actions-badge.atrox.dev` is recommended over [the one used here](https://github.com/BedrockStreaming/Statsd/pull/29/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1)

**Review by Oliboy50 — APPROVED** [review]
👏 😲

File: README.md (hunk 1 above)
Comment (jdecool) [reply README.md:1]: `https://github.com/BedrockStreaming/RedisMock/actions/workflows/ci.yml/badge.svg` is the official Github URL to display the status badge.

`actions-badge.atrox.dev` is an external project which made the same thing and was created before Github provide the feature.

So I suggest to use the _official_ link.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Continuous Integration
+on: [push]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        version: [ '7.1', '7.2', '7.3', '7.4', '8.0' ]
```
Comment (jdecool) [inline .github/workflows/ci.yml:10]: Maybe we can add `8.1` if the code is compatible ?

```suggestion
        version: [ '7.1', '7.2', '7.3', '7.4', '8.0', '8.1' ]
```

**Review by jdecool — APPROVED** [review]

**Review by jdecool — APPROVED** [review]

**Pull request merged**